prow/plank/controller_test.go

```diff
@@ -28,7 +28,7 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
this is so annoying, @fejta can we find a way to avoid having them when gofmt-ing?